org.freeorion.FreeOrion.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.freeorion.FreeOrion
+command: freeorion
+
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+
+modules:
```

Comment:
I'm sure there are at least static libraries and development headers to clean up.